app/src/sharedTest/java/org/oppia/app/home/ContinuePlayingActivityTest.kt

```diff
@@ -1,24 +1,304 @@
 package org.oppia.app.home
 
+import androidx.recyclerview.widget.RecyclerView
 import androidx.test.core.app.ActivityScenario
 import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
-import org.hamcrest.CoreMatchers.containsString
+import androidx.test.rule.ActivityTestRule
+import org.hamcrest.Matchers.containsString
+import org.hamcrest.Matchers.not
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.oppia.app.R
+import org.oppia.app.home.continueplaying.ContinuePlayingActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+import org.oppia.app.story.StoryActivity
+import org.oppia.app.utility.EspressoTestsMatchers.withDrawable
+import org.oppia.app.utility.OrientationChangeAction.Companion.orientationLandscape
 
 /** Tests for [ContinuePlayingActivity]. */
 @RunWith(AndroidJUnit4::class)
 class ContinuePlayingActivityTest {
 
+  @get:Rule
+  var activityTestRule: ActivityTestRule<ContinuePlayingActivity> = ActivityTestRule(
+    ContinuePlayingActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Before
+  fun setUp() {
+    Intents.init()
+  }
+
+  @Test
+  fun testContinuePlayingActivity_hasSomeOngoingPromotedStories() {
+    ActivityScenario.launch(ContinuePlayingActivity::class.java).use {
+      onView(
+        atPositionOnView(
+          R.id.ongoing_story_recycler_view,
+          0,
+          R.id.section_title_text_view
+        )
+      ).check(
+        matches(
+          withText(containsString("Played within the"))
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testContinuePlayingActivity_recyclerViewItem0_doesNotShowSectionDivider() {
+    ActivityScenario.launch(ContinuePlayingActivity::class.java).use {
+      onView(
+        atPositionOnView(
+          R.id.ongoing_story_recycler_view,
+          0,
+          R.id.divider_view
+        )
+      ).check(
+        matches(
+          not(isDisplayed())
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testContinuePlayingActivity_recyclerViewItem0_showsLastWeekSectionTitle() {
```

Comment:
Isn't this test redundant with testContinuePlayingActivity_hasSomeOngoingPromotedStories? Suggest keeping one of them.

Reply:
Removed `testContinuePlayingActivity_hasSomeOngoingPromotedStories `